symplyphysics/laws/optics/linear_magnification_from_distance_to_object_and_distance_to_image.py

```diff
@@ -0,0 +1,40 @@
+from sympy import (Eq, solve, S)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+                           validate_output, dimensionless, convert_to)
+
+# Description
+## Magnification, in optics, the size of an image relative to the size of the object creating it.
+## Depending on the position of the object in relation to the lens, the linear dimensions of the image change.
+
+
+## Law: M = f / d
+## Where:
+## M is linear magnification produced by lense
+## f is distance from lens to image
+## d is distance from lens to object
+
+## Conditions
+## If the image is inverted, then a plus sign is placed before the value f and a minus sign if the image is straigth.

```

Comment:
It's `straight`.
And I want you to describe signs in detail.

Reply:
![Screenshot_4](https://github.com/blackyblack/symplyphysics/assets/39147553/4f03f764-a8db-4254-913a-72927178dec1)
Before that, I had the right condition, I don't quite understand what else needs to be written in more detail, because everywhere the explanations for this formula are just like that.